src/components/ServiceDetails/AboutService/ServiceDemo/Purchase/ExpiredSession/GeneralAccountWallet/CreateWalletPopup/Summary/index.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { withStyles } from "@material-ui/styles";
+import Typography from "@material-ui/core/Typography";
+import { withRouter } from "react-router-dom";
+
+import StyledButton from "../../../../../../../../common/StyledButton";
+import { useStyles } from "./styles";
+import StyledTable from "../../../../../../../../common/StyledTable";
+import InfoIcon from "@material-ui/icons/Info";
+import Routes from "../../../../../../../../../utility/constants/Routes";
+
+const Summary = ({
```

Comment:
I think using props and destructuring within the component will be cleaner here.

Reply:
done https://github.com/singnet/snet-dapp/pull/402/commits/f418fa0f165d6a45e57905d425915ec9e01963bc